pkg/cmd/roachtest/tests/cluster_to_cluster.go

```diff
@@ -435,6 +602,28 @@ func registerClusterToCluster(r registry.Registry) {
 			additionalDuration: 10 * time.Minute,
 			cutover:            5 * time.Minute,
 		},
+		{
+			name:               "c2c/UnitTest",
```

Comment:
I wonder if we perhaps want to disable this one by default just because I imagine it is mostly useful for local testing?

Reply:
Done.